smartmodule/cargo_template/Smart.toml

```diff
@@ -2,9 +2,9 @@
 name = "{{project-name}}"
 group = "{{project-group}}"
 version = "0.1.0"
-package_version = "0.1"
+apiVersion = "0.1.0"
 description = "{{project-description}}"
 license = "Apache-2.0"
 
-[cargo]
-profile = "release"
+[params]
+
```

Comment:
These changes in the Smart.toml file were needed to make the toml read of the file work. Toml seems to interpret the field api_version as apiVersion when serialized. There is probably a way to force it to "api_version" if desired.

Added a unit test to load the template Smart.toml too